x/claim/keeper/claim.go

```diff
@@ -51,17 +53,21 @@ func (k Keeper) ClawbackAirdrop(ctx sdk.Context) error {
 		if err != nil {
 			return err
 		}
-		seq, err := k.accountKeeper.GetSequence(ctx, addr)
-		if err != nil {
-			return err
-		}
-		if seq == 0 {
-			osmoBal := k.bankKeeper.GetBalance(ctx, addr, "uosmo")
-			ionBal := k.bankKeeper.GetBalance(ctx, addr, "uion")
-			err = k.distrKeeper.FundCommunityPool(ctx, sdk.NewCoins(osmoBal, ionBal), addr)
+
+		acc := k.accountKeeper.GetAccount(ctx, addr)
+		if acc != nil {
```

Comment:
I'm trying to make sure I understand the prior bug. Why would it clawback from all accounts? (Or were the addresses just wrong)

Also is there any expected scenario where acc would be nil, or this is just in case something unexpected occurs?

Reply:
No, it was doing the opposite.  It wasn't clawing back from any accounts.
See original testcases:
https://github.com/osmosis-labs/osmosis/blob/cf7ba91f7bd74e6b9e7d3559f5e0e0ff9da87b8b/x/claim/keeper/claim_test.go

It's cause the function was exiting early, b/c as soon as it saw an account that wasn't in the account store it would error.

Reply:
Ahh that makes sense